test/float8/test_compile.py

```diff
@@ -39,6 +44,7 @@
 from torch._dynamo.test_case import TestCase as DynamoTestCase
 from torch._dynamo.testing import CompileCounterWithBackend
 
+# TODO(future PR): standardize IS_H100 with the rest of the codebase
```

Comment:
we want is SM89 for testing cublas matmuls, since in theory we have hardware with that capability on CI/CD

Reply:
yeah I meant the variable name, not what it's checking